tests/fastai/test_fastai_model_export.py

```diff
@@ -40,20 +39,22 @@ def fastai_model():
     iris = datasets.load_iris()
     X = pd.DataFrame(iris.data[:, :2], columns=iris.feature_names[:2])
     y = pd.Series(iris.target, name="label")
-    data = (
-        TabularList.from_df(pd.concat([X, y], axis=1), cont_names=list(X.columns))
-        .split_by_rand_pct(valid_pct=0.1, seed=42)
-        .label_from_df(cols="label")
-        .databunch()
+    dl = TabularDataLoaders.from_df(
+        df=pd.concat([X, y], axis=1), cont_names=list(X.columns), y_names="label"
     )
-    model = tabular_learner(data, metrics=accuracy, layers=[3])
+    model = tabular_learner(dl, metrics=accuracy, layers=[3])
     model.fit(1)
     return ModelWithData(model=model, inference_dataframe=X)
 
 
 @pytest.fixture
 def model_path(tmpdir):
-    return os.path.join(str(tmpdir), "model")
+    path = os.path.join(str(tmpdir), "model")
+    # Seems like Windows absolute path are not currently supported by MLFlow
+    # then they are transformed to relative path
+    if path.lower().startswith("c:"):
+        path = path[2:]
```

Comment:
do we need support run test on windows ? the 2 line logic looks weird, could it start with `d:` ?

Reply:
No, we don't need to.